libraries/ESP8266WiFi/src/ESP8266WiFiSTA.cpp

```diff
@@ -127,6 +127,7 @@ wl_status_t ESP8266WiFiSTAClass::begin(const char* ssid, const char *passphrase,
     }
 
     conf.threshold.rssi = -127;
+    conf.open_and_wep_mode_disable = true;

```

Comment:
Maybe this new call should depend on whether passphrase is empty?

Reply:
Yes, but wep should not be left aside even if it is obsolete. People sometimes need to experiment with old hardware and at the same time are living in open country.